components/atom/panel/src/index.scss

```diff
@@ -0,0 +1,62 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+
+$bgc-atom-panel: color-variation($c-gray, -2) !default;
+$bgc-panels: (
+  canvas $c-white,
+  contrast $c-black,
+  accent $c-accent,
+  corporate $c-primary,
+  base color-variation($c-gray, 3),
+  default $c-gray-lightest,
+  highlight color-variation($c-accent, 4)
+) !default;
+$c-atom-panel: $c-black !default;
+$horizontal-alignments-atom-panel: ('left', 'center', 'right') !default;
+$vertical-alignments-atom-panel: ('top', 'center', 'bottom') !default;
+
+.sui-atom-panel {
+  &-content {
+    position: relative;
+  }
+
+  @each $type in $bgc-panels {
+    $key: nth($type, 1);
+    $value: nth($type, 2);
```

Comment:
This looks like an smart solution, but looks a bit weird to me. 
i'm more used to see something like:

```scss
$bgc-panels: (
 'canvas': $c-white,
 'contrast': $c-black,
 'accent': $c-accent
) !default;
 
.sui-atom-panel {
  @each $type, $attr in $bgc-panels {
    &--#{$type} {
      @include bgcolor-overlay-opacity($attr);
    }
  }
}
```

what do you think about that? it's a bit more clear?

Reply:
Yes, I like it,  is much cleaner and clear than before.